host/host_linux.go

```diff
@@ -347,7 +347,7 @@ func PlatformInformation() (platform string, family string, version string, err
 		family = "debian"
 	case "fedora":
 		family = "fedora"
-	case "oracle", "centos", "redhat", "scientific", "enterpriseenterprise", "amazon", "xenserver", "cloudlinux", "ibm_powerkvm":
+	case "oracle", "centos", "redhat", "scientific", "enterpriseenterprise", "amazon", "xenserver", "cloudlinux", "ibm_powerkvm", "rocky":
```

Comment:
We should probably add more exhaustive RHEL derivatives distro from (for example) https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux_derivatives